cypress/integration/visual-test/toggle-test.spec.js

```diff
@@ -0,0 +1,21 @@
+import { baseURL, checkEyes } from '../../util';
+describe('Visual Regression Testing', () => {
+  before(() => {
+    cy.eyesOpen({
+      appName: 'Clarity',
+      testName: 'toggles',
+      browser: { width: 1024, height: 768, name: 'chrome' },
+    });
+  });
+  after(() => {
+    cy.eyesClose();
+  });
+  it('check toggle-button', () => {
+    cy.visit(baseURL + 'toggles');
+    checkEyes('toggle off');
```

Comment:
This page is very long, can we have it snapshot the whole layout? It seems to crop at 1024.

Reply:
There is a problem here. The page's container is set to have a height of `100vh` which is stopping Applitools to take the screenshot of the whole thing.
I guess the whole page screenshot will need some design + code changes

Reply:
Ok, that is something we'll have to look into. Do you mind putting a `@TODO` comment in here about it?